sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -101,16 +105,63 @@ def __init__(self, name):
 
         controller_list_param = rospy.get_param("/move_group/controller_list")
 
+        robot_description = rospy.get_param("/robot_description")
+        self._joint_limits = dict()
```

Comment:
better to have this self._joint_limits = {} to be compliant with the new linter (you have also a dictionary init below done with {} style :)